components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/Constants.java

```diff
@@ -38,6 +38,7 @@
     public static final String PREFIX_LOGIN = "/login";
     public static final String PREFIX_LOGOUT = "/logout";
     public static final String PREFIX_SERVER_DATA = "/server";
+    public static final String PREFIX_LOG_FILES = "/log-files";
```

Comment:
can't we simply say as /logs and make the query param also as "file" instead of file name .. doesn't that imply file name